pyrogram/types/messages_and_media/message.py

```diff
@@ -2732,32 +2732,16 @@ async def forward(
                     disable_notification=disable_notification,
                     schedule_date=schedule_date
                 )
-
-                if self.photo:
-                    file_id = self.photo.file_id
-                    file_ref = self.photo.file_ref
-                elif self.audio:
-                    file_id = self.audio.file_id
-                    file_ref = self.audio.file_ref
-                elif self.document:
-                    file_id = self.document.file_id
-                    file_ref = self.document.file_ref
-                elif self.video:
-                    file_id = self.video.file_id
-                    file_ref = self.video.file_ref
-                elif self.animation:
-                    file_id = self.animation.file_id
-                    file_ref = self.animation.file_ref
-                elif self.voice:
-                    file_id = self.voice.file_id
-                    file_ref = self.voice.file_ref
-                elif self.sticker:
-                    file_id = self.sticker.file_id
-                    file_ref = self.sticker.file_ref
-                elif self.video_note:
-                    file_id = self.video_note.file_id
-                    file_ref = self.video_note.file_ref
-                elif self.contact:
+                file_id, file_ref = None, None
```

Comment:
Please revert this, it has nothing to do with dice. You can open a new PR with your proposed changes (the relevant code can now be found in `async def copy`).